mitodl/ocw-hugo-themes
# Hash static assets for Cache-Control headers

# What are the relevant tickets?
Partial solution to #1212 

# Description (What does it do?)
This PR:
1. Changes directory of `/assets/images/` to `/assets/static_shared/images/` to have a consistent `/static_shared/images/` in the server for using Hugo's FingerPrint
2. Uses Hugo's FingerPrint for all `static_shared` assets used in partials -- fonts & images
3. Hashes `mathjax` library using `webpack`.

# How can this be tested?
I had smoke tested this and the changes seem to be working fine.
Smoke test these 4 themes: course-v2, course-offline, www, www-offline. More steps (commands) can be found here: https://github.com/mitodl/ocw-hugo-themes/pull/1225
First, checkout to this branch.
1. Make sure there's no typo in the changes
2. Make sure the changes are working as they were before.
  2a. Make sure `mathjax` library works. (Use this PR for `mathjax` testing instructions https://github.com/mitodl/ocw-hugo-themes/pull/906)
  2b Make sure the fonts are working as before.
In general, just make sure all the icons, images, `js` functionalities, css, everything is working as before.
Also check videojs (online+offline), nanogallery2(online+offline). Use the course `16.885j-fall-2005` and go to `Related Resources` from left for an example.

# Additional Context
This is the PR for adding cache-control headers: https://github.com/mitodl/ol-infrastructure/pull/1857
